pkg/provisioner/resourcehandlers.go

```diff
@@ -131,6 +131,53 @@ func createObjectBucket(ob *v1alpha1.ObjectBucket, c versioned.Interface, retryI
 	return
 }
 
+func updateObjectBucket(ob *v1alpha1.ObjectBucket, c versioned.Interface, retryInterval, retryTimeout time.Duration) (result *v1alpha1.ObjectBucket, err error) {
+	logD.Info("updating ObjectBucket", "name", ob.Name)
+	result, err = c.ObjectbucketV1alpha1().ObjectBuckets().Update(context.TODO(), ob, metav1.UpdateOptions{})
+	if err != nil {
+		return nil, err
+	}
+	return result, err
+}
+
+func getObjectBucketFromClaimKey(key string, obc *v1alpha1.ObjectBucketClaim, c versioned.Interface, retryInterval, retryTimeout time.Duration) (ob *v1alpha1.ObjectBucket, err error) {
+	obName, err := objectBucketNameFromClaimKey(key)
+	if err != nil {
+		return nil, err
+	}
+
+	logD.Info("seeing if object bucket for claim exists", "expected object bucket name", obName)
```

Comment:
maybe clearer?
`	logD.Info("seeing if OB for OBC exists", "checking for OB name", obName)`